manifests/java.yml

```diff
@@ -112,682 +120,14 @@ tests/:
-      Test_RemoteConfigurationUpdateSequenceFeatures:
-        '*': v1.4.0
-        spring-boot-3-native: missing_feature (GraalVM. Tracing support only)
-      Test_RemoteConfigurationUpdateSequenceFeaturesNoCache: v1.4.0
-      Test_RemoteConfigurationUpdateSequenceLiveDebugging:
-        '*': v1.4.0
-        spring-boot-3-native: missing_feature (GraalVM. Tracing support only)
-      Test_RemoteConfigurationUpdateSequenceLiveDebuggingNoCache: missing_feature
-  test_the_test/:
-    test_decorators.py:
-      Test_NotReleased: missing_feature
-    test_json_report.py:
-      Test_Mock: v0.0.99
-  test_distributed.py:
-    Test_DistributedHttp: missing_feature
-  test_identify.py:
-    Test_Basic: missing_feature
-    Test_Propagate: missing_feature
-    Test_Propagate_Legacy: missing_feature
-  test_library_conf.py:
-    Test_HeaderTags: missing_feature
-    Test_HeaderTagsLongFormat: v0.102.0
-    Test_HeaderTagsShortFormat: v0.102.0
-  test_profiling.py:
-    Test_Profile:
-      akka-http: missing_feature (Endpoint not implemented)
-      spring-boot-3-native: missing_feature (Tracing-only)
-      vertx3: missing_feature (Endpoint not implemented)
-      vertx4: missing_feature (Endpoint not implemented)
-  test_sampling_rates.py:
-    Test_SamplingDecisions:
-      akka-http: missing_feature (Endpoint /sample_rate_route not implemented)
-      jersey-grizzly2: missing_feature (Endpoint /sample_rate_route not implemented)
-      ratpack: missing_feature (Endpoint /sample_rate_route not implemented)
-      resteasy-netty3: missing_feature (Endpoint /sample_rate_route not implemented)
-      vertx3: missing_feature (Endpoint /sample_rate_route not implemented)
-      vertx4: missing_feature (Endpoint /sample_rate_route not implemented)
-  test_scrubbing.py:
-    Test_UrlQuery: v0.107.1
-  test_semantic_conventions.py:
-    Test_MetricsStandardTags: v1.6.0
-  test_standard_tags.py:
-    Test_StandardTagsClientIp:
-      '*': v0.114.0
-      akka-http: missing_feature (No AppSec support)
-      spring-boot-3-native: missing_feature (GraalVM. Tracing support only)
-    Test_StandardTagsMethod: v0.102.0
-    Test_StandardTagsRoute:
-      '*': missing_feature
-      spring-boot: v0.102.0
-      spring-boot-jetty: v0.102.0
-    Test_StandardTagsStatusCode: v0.102.0
-    Test_StandardTagsUrl: v0.107.1
-    Test_StandardTagsUserAgent: v0.107.1
-  test_telemetry.py:
-    Test_DependencyEnable: v1.7.0
-    Test_Log_Generation: missing_feature
-    Test_MessageBatch: missing_feature
-    Test_Metric_Generation_Disabled: missing_feature
-    Test_Metric_Generation_Enabled: missing_feature
-    Test_ProductsDisabled:
-      '*': missing_feature
-      spring-boot-3-native: missing_feature (GraalVM. Tracing support only)
-    Test_Telemetry:
-      '*': v0.108.1
-      spring-boot-3-native: missing_feature (GraalVM. Tracing support only)
-    Test_TelemetryV2: missing_feature
+      Test_BlockingGraphqlResolvers:
+        "*": "missing_feature"
```

Comment:
You removed a large chunk of the java manifest file, I suppose iut's a typo ?

Reply:
this is fixed now